test/e2e/testcases.sh

```diff
@@ -22,7 +22,8 @@ TESTCASES[16]="skip_deps"
 TESTCASES[17]="helm_local"
 TESTCASES[18]="no_updates_exist"
 TESTCASES[19]="oci_catalog"
-TESTCASES[20]="headsonly_diff_with_target"
+TESTCASES[20]="oci_local_all"
+TESTCASES[21]="headsonly_diff_with_target"
```

Comment:
For this test, I was going to suggest using `targetCatalog` instead of `targetName`. 
WDYT?
The catalog used there is hosted on registry, is that correct?

Reply:
@sherine-k - yes sure I can look at that